### PR TITLE
s2433: context help on Data Citation (Publication) page

### PR DESCRIPTION
**Pull request by hjeeves**
Code review needs to include the s2433 branch of the wwebui respository - the cadc.contexthelp.js file was added to support context help using popovers.

### CODE REVIEW

File: src/main/webapp/css/citation.css [hunk 1]
```diff
@@ -204,6 +204,39 @@
...
+}
+
+.popover-info {
+  margin-top:10px;
+  margin-bottom:10px;
+  color:mediumblue;
+}
+
+.glyphicon-question-sign, .glyphicon-remove-circle:hover
+{
```
Comment (at88mph) [inline src/main/webapp/css/citation.css]: Consistency with curly braces.

File: src/main/webapp/js/citation.js [hunk 2]
```diff
@@ -83,48 +80,29 @@
...
-    }
-
-    function setAuthenticated() {
-      $('.doi_authenticated').removeClass('hidden')
-      $('.doi-not-authenticated').addClass('hidden')
-      initializeDoiTable()
-      attachListeners()
     }
+    //
+    //function setNotAuthenticated(errorMsg) {
```
Comment (at88mph) [inline src/main/webapp/js/citation.js]: Just delete this if it's not needed, please.

File: src/main/webapp/css/citation.css (hunk 1 above)
Comment (hjeeves) [reply src/main/webapp/css/citation.css]: ok

File: src/main/webapp/js/citation.js (hunk 2 above)
Comment (hjeeves) [reply src/main/webapp/js/citation.js]: right. forgot to prune that out after finding it not needed...

**Pull request merged**